templates/method.mustache

```diff
@@ -106,7 +106,7 @@
     {{/isQueryParameter}}
 
     {{#isPathParameter}}
-        path = path.replace('{{=<% %>=}}{<%&name%>}<%={{ }}=%>', `${parameters['{{&camelCaseName}}']}`);
+        path = path.replace('{{=<% %>=}}{<%&name%>}<%={{ }}=%>', `${encodeURIComponent(parameters['{{&camelCaseName}}'].toString())}`);
```

Comment:
Is the tostring() necessary?

Reply:
The toString is required in case the input is a number or such, as encodeURIComponent requires string input.

Reply:
Makes sense!